argo/common/suite_unit_test.go

```diff
@@ -128,7 +128,7 @@ var _ = Context("NamespacedNameFromString", Serial, func() {
 var _ = Context("RunCompletionEvent.String", func() {
 	artList := []Artifact{
 		{
-			Name:     "ArtifactName",
+			Name:     "Name",
```

Comment:
need to revert this, think this has been done during a different refactor and intellij has done its thing